packages/instrumentation-console/src/version.ts

```diff
@@ -0,0 +1 @@
+export const VERSION = '0.0.0';
```

Comment:
```suggestion
export const VERSION = '0.0.0-dev';
```

Suffix indicates a development version, according to semantic versioning.